analyticsShared/src/androidMain/kotlin/mealz/ai/SharedAnalytics.android.kt

```diff
@@ -0,0 +1,48 @@
+package ai.mealz.analytics
+
+import com.google.gson.Gson
+import io.ktor.client.HttpClient
+import io.ktor.client.plugins.DefaultRequest
+import io.ktor.client.request.post
+import io.ktor.client.request.setBody
+import io.ktor.http.ContentType
+import io.ktor.http.contentType
+import kotlinx.coroutines.CoroutineExceptionHandler
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+actual object SharedAnalytics: AbstractSharedAnalytics() {
+    private val coroutineHandler = CoroutineExceptionHandler { _, exception ->
+        println("Mealz error in Analytics $exception ${exception.stackTraceToString()}")
+    }
+    private val coroutineScope: CoroutineScope = CoroutineScope(Dispatchers.IO + coroutineHandler)
+
+    private val httpClient = HttpClient {
+        install(DefaultRequest)
+    }
+
+    private suspend fun HttpClient.postEvent(event: PlausibleEvent) {
+        println("will send event $event to $PLAUSIBLE_URL") // TODO: log and log levels task
+        val gson = Gson()
+        val jsonEvent = gson.toJson(event)
+        post(PLAUSIBLE_URL) {
+            contentType(ContentType.Application.Json)
+            setBody(jsonEvent)
```

Comment:
il y a un intérêt a garder Gson dans un constante ou le jsonEvent ?

```suggestion
            setBody(Gson().toJson(event))
```

Reply:
Très honnêtement je sais pas, j'ai suivi la doc

Reply:
bah c'est juset que quand tu créer une val tu a un adresse en mémoire , il y a moyen qu'a la compilation tout seul il te le vire , comme tu veux je pense que ça n'aporte pas plus d'info